src/data/content.json

```diff
@@ -63,7 +63,7 @@
         "allPlants": "todas las plantas",
         "experience": "Experiencias personales"
       },
-      "changeTo": "Wissel taal naar",
+      "changeTo": "Wissel taal",
```

Comment:
ik begrijp dat je 'naar' weghaald hebt zodat je twee knoppen kon weergeven, maar om te muggenziften had je in dat geval 'Cambiar el idioma a' ook moeten veranderen naar 'Cambiar el idioma'